test/Sentry.Tests/SentryClientTests.cs

```diff
@@ -191,9 +194,6 @@ public void CaptureEvent_BeforeEvent_RejectEvent_RecordsDiscard()
     {
         _fixture.SentryOptions.BeforeSend = _ => null;
 
-        var transport = Substitute.For<ITransport>();
-        _fixture.SentryOptions.Transport = transport;
-
```

Comment:
Note that with the background worker substituted already, there's no purpose in substituting the transport.  It will never get invoked.